modules/aws/kube-worker/role.tf

```diff
@@ -124,3 +124,36 @@ resource "aws_iam_role_policy_attachment" "worker_vpc_cni" {
   policy_arn = aws_iam_policy.worker_vpc_cni[0].arn
   role       = aws_iam_role.worker[0].name
 }
+
+data "aws_iam_policy_document" "worker_ccm" {
```

Comment:
it's not necessary to create IAM Policy for ccm here, the permission is duplicated with `data "aws_iam_policy_document" "worker" {
`